aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -590,6 +605,20 @@ module aptos_framework::stake {
         );
     }
 
+    /// Allows to change the beneficiary for the oeprator of the stake pool.
+    public fun set_beneficiary_for_operator(pool_signer: &signer, new_beneficiary: address) acquires Beneficiary {
+        let pool_address = signer::address_of(pool_signer);
+        assert_stake_pool_exists(pool_address);
+        if (!exists<Beneficiary>(pool_address)) {
+            move_to(pool_signer, Beneficiary { beneficiary: new_beneficiary });
+        }
+        else {
```

Comment:
nit: Put else on the previous line } else

Reply:
This function is removed from the module.